index.js

```diff
@@ -100,40 +114,71 @@ const serializers = {
   },
   ReleaseEvent: (item) => {
     return `🚀 ${capitalize(item.payload.action)} release ${toUrlFormat(
-      item.payload.release.name
-        ? item.payload.release.name
-        : item.payload.release.tag_name
+      item
     )} in ${toUrlFormat(item.repo.name)}`;
   },
 };
 
 Toolkit.run(
   async (tools) => {
-    // Get the user's public events
     tools.log.debug(`Getting activity for ${GH_USERNAME}`);
-    const events = await tools.github.activity.listPublicEventsForUser({
-      username: GH_USERNAME,
-      per_page: 100,
-    });
+
+    let page = 1;
+    let events = [];
+
+    while (events.length < MAX_LINES) {
+      let data = [];
+
+      try {
+        // Get the user's public events
+        const resp = await tools.github.activity.listPublicEventsForUser({
+          username: GH_USERNAME,
+          per_page: 100,
+          page: page,
+        });
+
+        data = resp.data;
+      } catch (err) {
+        // Catch any HTTP errors. Especially because the API pagination is
+        // limited and throws an error when reaching the end
+        tools.log.info(err);
```

Comment:
```suggestion
        tools.log.info(err.message);
```

Let's log the message alone without the internal stack trace.